### PR TITLE
Fix test error on 0.6

### PR DESCRIPTION
**Pull request by yuyichao**

### CODE REVIEW

File: src/StatsFuns.jl [hunk 1]
```diff
@@ -228,6 +228,9 @@ include("misc.jl")
 include("rmath.jl")
 
 using .RFunctions
+if VERSION >= v"0.6.0-dev.2767"
```
Comment (ararslan) [inline src/StatsFuns.jl]: The version check isn't necessary since SpecialFunctions supports 0.5.

**Comment by ararslan** [comment]
Thanks as always, Yichao! While you're in here, if you don't mind, could you remove the 0.4 badge and add `[StatsFuns](http://pkg.julialang.org/?pkg=StatsFuns)` to the README after the 0.5 badge?

File: src/StatsFuns.jl (hunk 1 above)
Comment (yuyichao) [inline src/StatsFuns.jl]: it's needed or there was a test error.

Comment (ararslan) [inline src/StatsFuns.jl]: What was the test error?

Comment (yuyichao) [inline src/StatsFuns.jl]: Conflict import. You can check CI log. Basically SpecialFunctions isn't precise about importing/extending functions.

**Pull request merged**